_posts/2021-09-27-solidity-0.8.8-release-announcement.md

```diff
@@ -0,0 +1,177 @@
+
+The smallest and the largest value of an [enum
+type](https://docs.soliditylang.org/en/latest/structure-of-a-contract.html?#enum-types) can now be
+accessed using `type(E).min` and `type(E).max` respectively.
+
+```solidity
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity ^0.8.8;
+
+enum AuctionState {Created, InProgress, Completed}
+
+// returns AuctionState.Created
+function min() pure returns (AuctionState) {
+    return type(AuctionState).min;
+}
+
+// returns AuctionState.Completed
+function max() pure returns (AuctionState) {
+    return type(AuctionState).max;
+}
+```
+
+### Reading From Immutable Variables
+
+Immutable variables can be read at construction time once they are initialized.
+
+```solidity
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity ^0.8.8;
+
+interface IMockInterface {
+    function getValue() external returns(uint);
+}
+
+contract ImmutableExample {
+    uint immutable value;
+
+    event setValue(uint);
+
+    constructor(IMockInterface mockContract) {
+        // Sets the immutable variable.
+        value = mockContract.getValue();
+        // Reads the immutable variable.
+        emit setValue(value);
+    }
+}
+```
+
+As you probably know, it is also possible to initialize immutable variables
+at the point of their declaration, for example `uint immutable deployTime = block.timestamp;`.
+Because the order of initialization is not
+always obvious, such immutable variables are only considered initialized once
+the constructor is executed. This means you can read from such variables
+inside the constructor but you cannot use them to initialize other
+immutable variables at the point of their declaration.
+
+
+### Import Include Paths
+
+We added the new commandline option `--include-path` that hopefully makes it much easier to compile
+more complicated projects using multiple libraries. It is available in `solcjs` and `solc` both in legacy
+and standard-json mode. You can use it to specify
+directories where source files are looked up when `import` statements are resolved. If you use
+`npm` to install Solidity packages, they are usually placed in a subdirectory below
+`node_modules`. If you use imports of the form `import X from "@packagename/contracts/x.sol";`,
+then invoking the compiler as follows should properly resolve all imports without further work
+(the base path and the include paths will be added to the "allowed paths", for example):
+
+```bash
+solc contract.sol --base-path . --include-path node_modules/
+```
+
+The previously introduced option `--base-path` provides a path where the lookup is performed
+first. If it fails, the include paths are searched for the file.
+
+It is often a source of frustration that the Solidity compiler stores the full path
+to a source file as part of the metadata, which sometimes makes it more difficult to
+reproduce a compilation when not using the standard-json mode.
+
+If you use `--base-path` and `--include-path`, these paths will not be stored in
```

Comment:
It reads a bit as if "paths" was referring to file paths rather than paths passed to these options. So someone might think that file paths are not stored in metadata at all in this case. I'd reword it to make it completely clear:
```suggestion
If you use `--base-path` and `--include-path`, then only relative paths to files located inside them will be stored in
```